spec/controllers/verify/address_controller_spec.rb

```diff
@@ -30,4 +30,25 @@
       expect(response).to be_ok
     end
   end
+
+  describe '#create' do
+    it 'tracks the address delivery method event when phone is selected' do
+      stub_analytics
+      analytics_hash = {address_delivery_method: 'phone'}
+
+      expect(@analytics).to receive(:track_event).
+        with(Analytics::ADDRESS_DELIVERY_METHOD, analytics_hash)
+
+      post :create, params: {address_delivery_method: 'phone'}
+    end
+    it 'tracks the address delivery method event when usps is selected' do
```

Comment:
Very small nit here. It'd be nice to see some whitespace to separate these examples.